### DISCUSSION TITLE
Dark Mode and Rounded Corner

### DISCUSSION THREAD

**Discussion by Mohsyn** [WindowTop/WindowTop-App#336] [Q&A]
Why is the tiny little bit in all windows with rounded corner shown as white when dark mode is applied ?
I am 
![image](https://github.com/WindowTop/WindowTop-App/assets/51590078/b02f7ac1-e8fb-4332-8096-9a147b1f0cd6)
using 5.22.5 on windows 11

**Comment by gileli121** [comment]
Hello, this is a known minor issue.
When I developed, it was in Windows 10 days, when Windows did not have rounded corners, so such an issue happened.
The current implementation has an issue with round corners that cause this issue.

You will not notice it if the window is maximized because the rounded corners become square when it is maximized.

**Comment by gileli121** [comment]
Created https://github.com/WindowTop/WindowTop-App/issues/337
